src/index.html

```diff
@@ -5,17 +5,23 @@
     <title>EFIS Editor</title>
     <base href="/" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <link rel="icon" href="favicon-48.png" sizes="48x48" type="image/png" />
-    <link rel="icon" href="favicon.svg" sizes="any" type="image/svg+xml" />
-    <link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png" />
+
+    <link rel="icon" href="favicon.ico" sizes="48x48" />
+    <link rel="icon" type="image/png" href="favicon-48.png" sizes="48x48" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" sizes="any" />
+    <link rel="apple-touch-icon" href="apple-touch-icon.png" />
```

Comment:
```suggestion
    <link rel="icon" href="favicon.ico" sizes="16x16" />
    <link rel="icon" type="image/png" href="favicon-48.png" sizes="48x48" />
    <link rel="icon" type="image/svg+xml" href="favicon.svg" sizes="any" />
    <link rel="apple-touch-icon" href="apple-touch-icon.png"  sizes="180x180" />
```

I think the sizes should be corrected, right?

Also, does it make sense to add 192/512/512-nomask icons? Honest question.

Reply:
What I learned is that there's a surprising amount of complexity in favicons :) different browsers behave differently and change behavior over the years, and PWA just adds more to that complexity.

wrt the `apple-touch-icon` size, I had it originally, and then safari tries to fetch a different file called `apple-touch-icon-precomposed.png` because it actually wants a different size :) a quick web search told me I could remove the size and Safari would use this one.

wrt favicon.ico, that file contains 3 sizes - 16x16, 32x32 and 48x48 - I looked up which one should be declared, and turns out it's all of them, so updated that.

> Also, does it make sense to add 192/512/512-nomask icons? Honest question.

My understanding is that no browser will use the 192 or 512 icons unless they're installing the PWA on the OS, and some browsers will fetch *all* `link rel="icon"` icons proactively, so it seems better to have those only on the manifest.

Reply:
All right, thank you for the explanation!

In IntelliJ, only 16x16 was visible, but that was a red herring. Midnight Commander shows all three correctly via ImageMagic ;-)